### PR TITLE
Set up for Heroku

### LINKED ISSUE #31: Deploy the application(Lets go Live)
The application should be deployed to a Paas (Platform as a service) like [Heroku](https://www.heroku.com/platform).
Going live will allow for the initial user testing-feedback phase.
Read through the official docs for guidance.

### PR DESCRIPTION
**Pull request by blakes24**
### What does this PR do?
- This PR configures the app to deploy on Heroku
### How should this be manually tested?
- Go to [https://dive-atlas.herokuapp.com/](https://dive-atlas.herokuapp.com/)

### CODE REVIEW

**Review by alexkayabula — CHANGES_REQUESTED** [review]
Good work @blakes24 but I am proposing that we create a separate `manage.py` file that caters for the `production environment` and we leave the `run.py` file for development. What do you think ?

**Comment by blakes24** [comment]
Makes sense.  `manage.py` file added.

File: config.py [hunk 1]
```diff
@@ -21,12 +20,13 @@ class TestingConfig(Config):
...
 
     TESTING = True
+    WTF_CSRF_ENABLED = False
     SQLALCHEMY_DATABASE_URI = os.environ.get('TEST_DATABASE_URL') or 'postgresql:///dive_test'
 
 class ProductionConfig(Config):
     """Configurations for Production Environment."""
 
-    SQLALCHEMY_DATABASE_URI = os.environ.get('PROD_DATABASE_URL')
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL')
```
Comment (alexkayabula) [inline config.py:29]: @blakes24 kindly update this change in the readme file.

**Comment by blakes24** [comment]
README updated

**Review by alexkayabula — APPROVED** [review]
Thank you.

**Pull request merged**